CHANGELOG.md

```diff
@@ -2,56 +2,50 @@
 Notable changes to the framework should be documented here
 
 3.14.0:
-	- - fix isvariable syntax error in update_def.cf (CFE-2953)
-	- - fixup FR push_over_rsync for FR i2 (ENT-4587)
-	- Add path support for setfacl
-	- Add path support for timedatectl and journalctl (CFE-3013)
-	- Add trailing slash to access promises expecting directories
+	- Fixed isvariable syntax error in update_def.cf (CFE-2953)
+	- Added path support for setfacl, timedatectl and journalctl (CFE-3013)
+	- Added trailing slash to access promises expecting directories
 	  (CFE-3024)
 	- Added scripts and templates for Federated Reporting (ENT-4473)
-	- Avoid requiring rpm python module to check zypper version
-	- Change cleanup consumer status SQL query (ENT-4365)
+	- rpm python module is no longer required to check zypper version
+	- Changed cleanup consumer status SQL query (ENT-4365)
 	- Conditioned use of curl for ec2 metadata cache on curl binary being executable
 	  (CFE-3049)
-	- Control cf-hub without modifying vendored policy (ENT-4269)
-	- Do not run DB maintenance tasks on a passive HA hub (ENT-4706)
-	- Do not suppress repair outcome for starting cf-monitord or cf-execd
+	- Added augments variables to control cf-hub (ENT-4269)
+	- Prevented DB maintenance tasks on a passive High Availability hub (ENT-4706)
+	- Repair outcome for starting cf-monitord or cf-execd is no longer suppressed
 	  (CFE-2964)
-	- Enforce restrictive permissions on hub install log (ENT-4506)
-	- Ensure that asynchronous query API semaphores are writable (ENT-4551)
-	- Fix standalone_self_upgrade not triggering because of stale data
+	- Restrictive permissions on hub install log are now enforced (ENT-4506)
+	- Ensured that asynchronous query API semaphores are writable (ENT-4551)
+	- Fixed standalone_self_upgrade not triggering because of stale data
 	  (ENT-4317)
 	- Fixed maintenance policy for promise log cleanup to respect history_length_days
 	  (ENT-4588)
-	- Improve efficiency and error handling of user specified policy update bundle
+	- Improved efficiency and error handling of user specified policy update bundle
 	- Log version of Enterprise agent outside of state (ENT-4352)
-	- Package module for managing windows packages using msiexec (ENT-3719)
-	- Prevent inventorying un-expanded free memory from cf-monitord
-	- Prevent mon.value_mem_total from being inventoried if not defined
```

Comment:
Why remove mention of this small improvement?

Reply:
I tried combining the 2 lines into 1 line which makes sense to me (pretending to be a normal user):

```
Prevented inventorying un-expanded memory values from cf-monitord (ENT-4522)
```

(Below)

Reply:
makes sense.